pkgs/games/xbill/default.nix

```diff
@@ -17,6 +17,24 @@ stdenv.mkDerivation rec {
     url = "http://www.xbill.org/download/${pname}-${version}.tar.gz";
     sha256 = "13b08lli2gvppmvyhy0xs8cbjbkvrn4b87302mx0pxrdrvqzzz8f";
   };
+  
+  postInstall = ''
+    mkdir -p $out/share/applications $out/share/pixmaps
+    cp pixmaps/icon.xpm $out/share/pixmaps/xbill.xpm
+    cat > $out/share/applications/xbill.desktop <<EOF
+    [Desktop Entry]
+    Type=Application
+    Name=XBill
+    Comment=Get rid of those Wingdows viruses!
+    Comment[de]=Werden Sie diese Wingdows-Viren los!
+    Comment[it]=Fai piazza pulita dai virus Wingdows!
+    Icon=xbill
+    Exec=$out/bin/xbill
+    Terminal=false
+    Categories=Game;ArcadeGame;
+    Keywords=Game;Bill;Viruses
+    EOF
```

Comment:
please switch to using makeDesktopItem. examples in nixpkgs, and see the builder

Reply:
Thank you, did so!